parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -186,6 +186,7 @@ apt_get_dist_upgrade() {
   apt_dist_upgrade_output=/tmp/apt-get-dist-upgrade.out
   for i in $(seq 1 $retries); do
     wait_for_apt_locks
+    export DEBIAN_FRONTEND=noninteractive
```

Comment:
Added this because postfix + dist-upgrade was adding a package dependency that yielded an interactive, blocking message in the dist-upgrade operation. Glad we caught this in testing. :)

Reply:
I thought we weren't installing postfix?

Reply:
This PR is only changing E2E besides this change so why is this change needed?

Reply:
Sorry, yes!

Actually, it's possible my testing stumbled into a bug w/ dist-upgrade.

Basically, we need to be ensuring all apt operations in scripts have `DEBIAN_FRONTEND=noninteractive` context